pkg/handlers/lazyjira.go

```diff
@@ -4,11 +4,13 @@ import (
 	"errors"
 	"fmt"
 	tea "github.com/charmbracelet/bubbletea"
+	"github.com/matthewrobinsondev/lazyjira/pkg/clients"
```

Comment:
we should probs rename the gomod now we've moved repo's, for a different ticket tho